### PR TITLE
Added Planck keymap: jcorrado

### PR DESCRIPTION
**Pull request by jcorrado**
PR to add my Planck keymap, which is based on the way I use the Kinesis Advantage: modifiers under thumbs, minimal contortion for modifier+layer combos, with either hand.

### CODE REVIEW

File: keyboards/planck/keymaps/jcorrado/keymap.c [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
```
Comment (noroadsleft) [inline keyboards/planck/keymaps/jcorrado/keymap.c]: Lines 18-19 should be replaced with `#include QMK_KEYBOARD_H`.

File: keyboards/planck/keymaps/jcorrado/keymap.c [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | CapLk/ | Z      | X      | C      | V      | B      | N      | M      | <      | >      | ?      | Shift/ |
+ * | Shift  |        |        |        |        |        |        |        | ,      | .      | /      | Enter  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | Toggle | ~      | {pipe} | Alt    | Ctrl   | Space  | Space  | Ctrl   | Alt    | {      | }      |        |
+ * | Num    | `      | \      |        |        |        |        |        |        | [      | ]      |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ */
+
+[_QWERTY] = {
```
Comment (noroadsleft) [inline keyboards/planck/keymaps/jcorrado/keymap.c]: All your layers should be formatted to use an appropriate LAYOUT macro, as seen here:

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/keyboards/planck/keymaps/default/keymap.c#L58-L63

**Comment by jcorrado** [comment]
Thanks for your help, @noroadsleft  Closing this PR and submitting a new one momentarily.

**Comment by noroadsleft** [comment]
You don't actually have to close PRs to make further changes.

Make the additional changes on the same branch that the PR is using as source, and push to GitHub. Your PR will update automatically.

**Pull request closed without merging**